### ISSUE TITLE
Vue 2 support

### ISSUE THREAD

**Issue by gitgkk** [vuejs/vue#13082]
### What problem does this feature solve?
Continue to create safe and secure applications with Vue 2.x

### What does the proposed API look like?
I've seen the statements and the roadmap here:
https://github.com/vuejs/vue/projects/6#card-16117796
https://v2.vuejs.org/lts/

I have a large application made with Vue 2 and Vuetify 2. I cannot migrate the application until Vuetify 3 components are ready and mature. I'd like to request to continue support for Vue 2 till the ecosystem is ready to allow migration. The migration is a long and expensive process. Clients don't like buggy applications. Being a small bootstrapped startup, we can't afford the expensive paid support.
So, I request for the support to be extended for another 6 months till June 2024, by then Vuetify 3 will be ready. 

Can I know if you will be able to extend the support for Vue 2.x in favor of people who are in similar situation like me?

Thanks,
Gajendra Kashyap

**Comment by MarcelGeo** [comment]
We are in same situation. Our solution:

- update to Vue 2.7 with vite and pinia
- remove vuetify and choose other library with Vue 3 support
- upgrade to Vue 3 
In vuetify are problems with migration of data tables and others to Vue 3.

**Comment by gitgkk** [comment]
Hi Marcel, 
We have a small company and team, we can't afford to switch so easily with a large application. It is a lot of work. 

Thanks.

**Comment by MarcelGeo** [comment]
Yes.

Maybe first step:

update to Vue 2.7 with vite and pinia

will be good start :)

**Comment by yyx990803** [comment]
We announced the EOL plan 18 months ago when 2.7 was released, and provided the promised 18 months of LTS. This 18 months was supposed to be the period for users to plan and execute a migration plan.

As an independent project providing all of our code for free, I think we have done what we could. We have limited bandwidth, and we need to focus what we have on improving Vue 3 for the future. Unfortunately, that means Vue 2 support will end as planned.
